pkg/metrics/metrics.jsx

```diff
@@ -771,14 +782,22 @@ class MetricsMinute extends React.Component {
                 { this.props.events.events.map(t => <span className="type" key={ t }>{ RESOURCES[t].event_description }</span>) }
                 <div className="details">
                     <time>{ timeformat.time(timestamp) }</time>
-                    {this.state.expanded && this.state.logsUrl && this.state.logs && this.state.logs.length ? <Button variant="link" isInline onClick={e => cockpit.jump(this.state.logsUrl)}>{_("View all logs")}</Button> : null}
+                    {this.state.expanded && this.state.logsUrl &&
+                        <Button variant="link" isInline onClick={e => cockpit.jump(this.state.logsUrl)}>
+                            { _("View detailed logs") }
+                        </Button>}
                 </div>
             </div>;
 
             let body = " "; // Cannot be false-y, otherwise table does not show '>'
             if (this.state.expanded) {
                 body = <div className="cockpit-log-panel">
-                    {this.state.logs === null ? _("Loading...") : this.state.logs.length === 0 ? _("No logs found") : this.state.logs}
+                    {this.state.logs === null
+                        ? _("Loading...")
+                        : this.state.logs.length === 0
+                            ? <span className="pf-u-py-md">{ _("No logs found") }</span>
```

Comment:
Whoops. In looking at it in action here, I think we actually want `-sm`. I'm sorry for suggesting `md` instead of `sm`.

Aside from that, it looks good to me!